playbooks/common/openshift-cluster/upgrades/pre/verify_cluster.yml

```diff
@@ -41,6 +41,16 @@
     - openshift_release is defined
     - not (openshift_release is version_compare(openshift_upgrade_target ,'='))
 
+  # Need client package to check gluster health when upgrading from containerized.
+  - name: Install openshift clients on first master
+    package:
+      name: "{{ openshift_service_type }}-clients{{ (openshift_pkg_version | default('')) | lib_utils_oo_image_tag_to_rpm_version(include_dash=True) }}"
```

Comment:
`The error was: 'openshift_service_type' is undefined` in https://openshift-gce-devel.appspot.com/build/origin-ci-test/pr-logs/pull/openshift_openshift-ansible/10783/pull-ci-openshift-openshift-ansible-310-gcp-major-upgrade/329